src/index.html

```diff
@@ -3,12 +3,52 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="moyoLogo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a class="nav_link is-active" href="#">apple</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="#">samsung</a>
+          </li>
+          <li class="nav_item">
+            <a class="nav_link" href="#">smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a
+            class="nav_link"
+            href="#"
+            data-qa="hover"
+            >laptops & computers</a>
```

Comment:
```suggestion
            <a
              class="nav_link"
              href="#"
              data-qa="hover"
            >
              laptops & computers
            </a>
```

Reply:
![image](https://github.com/mate-academy/layout_moyo-header/assets/143609932/045fbba2-5fe9-4469-a9c4-4142a4cc007b)
after git push origin develop comes error how to repair it?

Reply:
i`m use git command pull. done changes after that reupload this branch and now i`m waiting for a new rewiew